src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -247,3 +257,158 @@ def getStringRDD(self, varName, format):
         #    return rdd
         #except Py4JJavaError:
         #    traceback.print_exc()
+
+def getNumCols(numPyArr):
+	if numPyArr.ndim == 1:
+		return 1
+	else:
+		return numPyArr.shape[1]
+       
+def convertToJavaMatrix(sc, src):
+	src = np.asarray(src)
+	numCols = getNumCols(src)
+	numRows = src.shape[0]
+	arr = src.ravel().astype(np.float64)
+	buf = bytearray(arr.tostring())
+	return sc._jvm.org.apache.sysml.runtime.instructions.spark.utils.RDDConverterUtilsExt.convertPy4JArrayToMB(buf, numRows, numCols)
+	
+
+def convertToNumpyArr(sc, mb):
+	numRows = mb.getNumRows()
+	numCols = mb.getNumColumns()
+	buf = sc._jvm.org.apache.sysml.runtime.instructions.spark.utils.RDDConverterUtilsExt.convertMBtoPy4JDenseArr(mb)
+	return np.frombuffer(buf, count=numRows*numCols, dtype=np.float64)
+
+class mllearn:
+    # Or we can create new Python project with package structure
+    class LogisticRegression(Estimator):
+
+        def __init__(self, sqlCtx, penalty='l2', fit_intercept=True, max_iter=100, max_inner_iter=0, tol=0.000001, C=1.0, solver='newton-cg', transferUsingDF=False):
+            self.sqlCtx = sqlCtx
+            self.sc = sqlCtx._sc
+            self.log = self.sc._jvm.org.apache.sysml.api.ml.LogisticRegression("lr", self.sc._jsc.sc())
+            self.transferUsingDF = transferUsingDF
+            if penalty != 'l2':
+                raise Exception('Only l2 penalty is supported')
+            if fit_intercept:
+                self.icpt = 1
+            else:
+                self.icpt = 0
+            self.max_iter = max_iter
+            self.max_inner_iter = max_inner_iter
+            self.tol = tol
+            if C == 0:
+                raise Exception('C cannot be 0')
+            reg = 1/C
```

Comment:
As an alternative, we could use `from __future__ import division` to prevent this issue from creeping in.